contracts/minter-suite/Minters/MinterSEAV0.sol

```diff
@@ -0,0 +1,1186 @@
+     * this minter's purchase function, avoiding extra gas costs from the core contract's maxInvocations check.
+     * @param _projectId Project ID to set the maximum invocations for.
+     * @param _maxInvocations Maximum invocations to set for the project.
+     */
+    function manuallyLimitProjectMaxInvocations(
+        uint256 _projectId,
+        uint256 _maxInvocations
+    ) external {
+        _onlyArtist(_projectId);
+        // CHECKS
+        // ensure that the manually set maxInvocations is not greater than what is set on the core contract
+        uint256 maxInvocations;
+        uint256 invocations;
+        (invocations, maxInvocations, , , , ) = genArtCoreContract_Base
+            .projectStateData(_projectId);
+        require(
+            _maxInvocations <= maxInvocations,
+            "Cannot increase project max invocations above core contract set project max invocations"
+        );
+        require(
+            _maxInvocations >= invocations,
+            "Cannot set project max invocations to less than current invocations"
+        );
+        // EFFECTS
+        // update storage with results
+        projectConfig[_projectId].maxInvocations = uint24(_maxInvocations);
+        // We need to ensure maxHasBeenInvoked is correctly set after manually
+        // setting the local maxInvocations value.
+        projectConfig[_projectId].maxHasBeenInvoked =
+            invocations == _maxInvocations;
+
+        emit ProjectMaxInvocationsLimitUpdated(_projectId, _maxInvocations);
+
+        // for convenience, try to mint and assign a token to the project's
+        // next slot
+        _tryMintTokenToNextSlot(_projectId);
+    }
+
+    /**
+     * @notice Sets the minimum and maximum values that are settable for
+     * `durationSeconds` for all project configurations.
+     * Note that the auction duration is the initial duration of the auction,
+     * and does not include any extensions that may occur due to new bids being
+     * placed near the end of an auction.
+     * @param _minAuctionDurationSeconds Minimum auction duration in seconds.
+     * @param _maxAuctionDurationSeconds Maximum auction duration in seconds.
+     */
+    function updateAllowableAuctionDurationSeconds(
+        uint32 _minAuctionDurationSeconds,
+        uint32 _maxAuctionDurationSeconds
+    ) external {
+        _onlyCoreAdminACL(this.updateAllowableAuctionDurationSeconds.selector);
+        // CHECKS
+        require(
+            _maxAuctionDurationSeconds > _minAuctionDurationSeconds,
+            "Only max gt min"
+        );
+        require(_minAuctionDurationSeconds > 0, "Only min gt 0");
+        // EFFECTS
+        minAuctionDurationSeconds = _minAuctionDurationSeconds;
+        maxAuctionDurationSeconds = _maxAuctionDurationSeconds;
+        emit AuctionDurationSecondsRangeUpdated(
+            _minAuctionDurationSeconds,
+            _minAuctionDurationSeconds
```

Comment:
Suggested fix for copypaste error.

```suggestion
            _maxAuctionDurationSeconds
```

Reply:
⚠️ code smell since this wasn't caught in tests. Realized that quite a few events needed to be added to this minter's tests, so I've patch this and updated our tests to ensure all event values are as-expected in [bd8520b](https://github.com/ArtBlocks/artblocks-contracts/pull/504/commits/bd8520b582adf0709a3c23720767d36e4e6d5d63)